core/SymbolRef.h

```diff
@@ -75,6 +75,29 @@ class SymbolRef final {
         return _id;
     }
 
+    inline bool isClassOrModule() const {
+        return kind() == Kind::ClassOrModule;
+    }
+
+    inline bool isMethod() const {
+        return kind() == Kind::Method;
+    }
+
+    inline bool isFieldOrStaticField() const {
+        return kind() == Kind::Field;
+    }
+
+    inline bool isTypeArgument() const {
+        return kind() == Kind::TypeArgument;
+    }
+
+    inline bool isTypeMember() const {
+        return kind() == Kind::TypeMember;
+    }
```

Comment:
Knowing that we have an enum class for this kind now, I kind of want to start moving away from these `isFoo` helpers and towards forcing people to pattern match at the call sites (for exhaustiveness, because most people don't realize that they want to handle more symbol kinds than they start out handling).

Maybe i'll try that some time.

Reply:
I'm 👍 to that.